app/db/mongodb.py

```
@@ -27,6 +28,26 @@ def connect(self) -> None:
+                    ssh_host=settings.mongo_ssh_host,
+                    ssh_port=settings.mongo_ssh_port,
+                    ssh_user=settings.mongo_ssh_user,
+                    ssh_pkey=settings.mongo_ssh_pem_key_path,
+                    remote_bind_address=("127.0.0.1", port),
+                    local_bind_port=settings.mongo_local_bind_port,
+                )
+                # 터널을 통해 접속하므로 호스트와 포트 변경
+                host = "localhost"
+                port = settings.mongo_local_bind_port
+                logger.info(f"MongoDB SSH tunnel established on localhost:{port}")
+            except Exception as e:
```

Ruff (0.14.7)

Do not catch blind exception: `Exception`

(BLE001)